apps/E2E/src/CheckboxV1/specs/CheckboxV1.spec.win.ts

```diff
@@ -53,6 +53,22 @@ describe('CheckboxV1 Accessibility Testing', () => {
 
     await expect(await CheckboxV1PageObject.didAssertPopup()).toBeFalsy(CheckboxV1PageObject.ERRORMESSAGE_ASSERT);
   });
+
+  it('Set "required" prop. Validate "IsRequiredForForm" element attribute is true.', async () => {
+    await expect(
+      await CheckboxV1PageObject.compareAttribute(CheckboxV1PageObject._secondaryComponent, Attribute.IsRequiredForForm, 'True'),
```

Comment:
I believe "True" can be AttributeValue.true (for consistency across the package and type checking)

Reply:
Thanks! Didn't know it was available. Updated both spec files.